woocommerce/woocommerce-admin
# Add fancy placeholders for Marketing components

Add placeholders for 

- [x] RecommendedExtensionsItem.
- [x] KnowledgeBase.

Addresses https://github.com/woocommerce/woocommerce-admin/issues/5375.

### Accessibility




### Screenshots
![GIF animation of placeholders](https://user-images.githubusercontent.com/17435/99014000-0799d480-2552-11eb-80dd-cbfcfffc60e4.gif)
https://d.pr/i/uTkAtS

### Detailed test instructions:

1. Ex: Open page `http://wptest.local/wordpress/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`
2. Scroll down if needed,
3. If the content blinks too fast you can throttle network speed in dev tools.
4. Observe the "Recommended extensions" and "Knowledge base" sections, before the content is fully loaded.



### Changelog Note:
